html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML forms
+    </title>
+  </head>
+  <body>
+    <h1><strong>Coding Form!</strong></h1>
+    <form>
+      <div><label for="email">Email:</label></div>
+      <div><input type="text" name="email" size="30"></div>
+      <div><label for="option">Why is coding cool?</label></div>
+      <div>
+        <select name="Why is coding cool?">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <input type="radio" name="Favorite Language" value="HTML"> HTML
+        <input type="radio" name="Favorite Language" value="CSS"> CSS
+        <input type="radio" name="Favorite Language" value="JavaScript"> JavaScript
+      </div>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="signup"> Sign me up!
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea name="comments" cols="30" rows="10"></textarea>
```

Comment:
The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.